<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>571</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="select_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Edit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>191</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 219, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_des">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>230</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_wind">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>191</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 219, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wind_2">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>310</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_4">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>380</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_5">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>370</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp_2">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>270</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_des_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_6">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>380</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_2">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>370</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_des_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>191</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 219, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_day1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Day 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_day2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Day 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_day3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Day 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp_day2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_wind_day2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp_day3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_wind_day3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
